.github/workflows/release_gh_runner.yaml

```diff
@@ -0,0 +1,18 @@
+name: Release GitHub Runner
```

Comment:
"Launch Terraform Apply" or "Apply changes to infrastructure"

Reply:
ditto

Reply:
@Krusty93 also here `Release - GitHub Runner TF Validation` ?